src/lib/unit-conversion.js

```diff
@@ -4,7 +4,7 @@ const { isNull } = require('lodash');
 
 const cubicMetresToMegalitres = value => {
   if (isNull(value)) {
-    return null;
+    return 0;
```

Comment:
I'm not sure that we can do this because in certain cases, null and 0 mean two different things